### PR TITLE
Upgrade to Webpack2 compatibility

### PR DESCRIPTION
**Pull request by CoenWarmer**
Might not be the best implementation as I'm not super familiar with the Webpack Loader API, but it works and all tests pass :)

### CODE REVIEW

**Comment by CoenWarmer** [comment]
Also, this is my first Github PR, so please be gentle ;)

File: lib/swig-loader.js [hunk 1]
```diff
@@ -10,10 +10,13 @@ var loaderUtils = require('loader-utils'),
...
 module.exports = function(content) {
-	var plugin = this;
+	var plugin = assign(
+		{},
+		this
+	);
 	plugin.cacheable && plugin.cacheable();
-	plugin.query = loaderUtils.parseQuery(this.query);
-	plugin.resourceQuery = loaderUtils.parseQuery(this.resourceQuery);
+	plugin.query = JSON.parse(JSON.stringify(queryString.parse(plugin.query, { arrayFormat: 'bracket' })));
```
Comment (pmendelski) [inline lib/swig-loader.js]: Could you please extract `parseQuery` function.

Example:
```js
function parseQuery(query) {
  return JSON.parse(JSON.stringify(queryString.parse(query, { arrayFormat: 'bracket' })));
}
// ...
plugin.query = parseQuery(plugin.query);
plugin.resourceQuery = parseQuery(plugin.resourceQuery);
```

**Review by pmendelski — CHANGES_REQUESTED** [review]
Congratulations for your first GitHub PR. Well done :)

There is just one change to be applied - extract function. When it's done I will merge your pr and release a new version.

BTW. It would be also nice if you could squash your commits into a single one. Though commit history of this repo is not perfect, so in this case it's not a big problem ;)

**Comment by CoenWarmer** [comment]
Will do! Thanks :)

**Comment by CoenWarmer** [comment]
Seems that by squashing commits and force pushing I can't address your review comment through Github's UI anymore.. But it's in there! :)

**Comment by pmendelski** [comment]
Thanks for your PR! :) When I get home I will publish it.

**Comment by pmendelski** [comment]
Published [swig-loader v2.0.0](https://www.npmjs.com/package/swig-loader)

**Pull request merged**